### ISSUE TITLE
Page Stuck JS

### ISSUE THREAD

**Issue by jpcapone** [treyyoder/quakejs-docker#18]
After I run the docker container I attempt to join the server but the page is stuck on JS.  Anyone have any suggestions for remediation?

**Comment by jpcapone** [comment]
I found that if I go to http://www.quakejs.com/ and disconnect i can then join my local server successfully.  If anyone has any ideas regarding how to fix this please let me know.

**Comment by jpcapone** [comment]
Ok I confirmed my settings after reading a previous post which pointed me to the IP address used by the container.  changing it to the host IP address AND I forcing a reload of the page (CTRL+ R) in chrome everything worked like a charm.

**Comment by treyyoder** [comment]
Yeah, I just tested this and its broke:  Failed to execute 'open' on 'XMLHttpRequest': Invalid URL

Ill try to debug soon.

**Comment by m-kronberg** [comment]
Adding environment variables with the proper server up address and port fixes it for me

on docker-compose, for example:

        environment:
            - HTTP_PORT=8080
            - SERVER=192.168.2.110

issue is, at least in my version, entrypoint.sh expects ${SERVER} to be set.

**Comment by stebansaa** [comment]
> Yeah, I just tested this and its broke: Failed to execute 'open' on 'XMLHttpRequest': Invalid URL
> 
> Ill try to debug soon.

Any way to solve this?
